src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
Ah, I see. Yes, that will need to change. If you don't want to adopt the new syntax, you can just use what is currently available for inheritance:
```js
var FullWidthPopup = function(player, options) {
  vjs.Component.call(this, player, options);
  // ...
};
FullWidthPopup.prototype = Object.create(vjs.Component.prototype);
```

This is something that a migration plugin could plaster over with lots of helpful logs in the console.